rollup/test-resources.config.js

```diff
@@ -1,13 +1,9 @@
-// rollup-iife.config.js
-import resolve from 'rollup-plugin-node-resolve'
-import babel from 'rollup-plugin-babel'
-import commonjs from 'rollup-plugin-commonjs'
-import { terser } from 'rollup-plugin-terser'
 import replace from '@rollup/plugin-replace'
-import packageJson from '../package.json'
-import cleaner from 'rollup-plugin-cleaner'
-import strip from '@rollup/plugin-strip'
-import ts from "rollup-plugin-ts";
+import fs from 'fs'
+import path from 'path'
+import config from './dist.config.js'
+
+const packageJson = JSON.parse(fs.readFileSync(path.resolve('package.json'), { encoding: 'utf-8' }))
```

Comment:
Wouldn't `import packageJson from '../package.json' assert { type: "json" }` be more concise? You could also get rid of the `fs` and `path` imports.

Reply:
What do you mean? What are the situations it does not work?

Reply:
Bumping it ⬆️

Reply:
` ExperimentalWarning: Import assertions are not a stable feature of the JavaScript language. Avoid relying on their current behavior and syntax as those might change in a future version of Node.js.`

Reply:
👍